sdcm/cluster.py

```diff
@@ -3825,10 +3825,24 @@ def create_prometheus_snapshot(self, node):
             raise PrometheusSnapshotErrorException(result)
 
     def get_prometheus_snapshot(self, node):
```

Comment:
rename `get_prometheus_snapshot` -> `get_prometheus_data`
since the method get the data **by** snaphot feature of the database/by copying data dir

Reply:
add a log message when getting the data using Prometheus Snapshot and when using the data from the Prometheus data dir

Reply:
fixed